doc/guide.md

```diff
@@ -0,0 +1,263 @@
+shape that has the most space around it.
+
+![Motivation](images/motivation.png)
+
+This circle, known as the maximum inscribed circle, will always have its center lie on the
+[medial axis](https://en.wikipedia.org/wiki/Medial_axis) of the shape. The medial axis can be thought of as the bone
+structure of a shape, where every point on the medial axis has at least two equidistant closest points on the shape.
+Note, the following example is a manual approximation and is not 100% accurate, particularly in the curved segments.
+
+![Medial Axis](images/medial-axis.png)
+
+A [Voronoi diagram](https://en.wikipedia.org/wiki/Voronoi_diagram) over the discretized edges of the shape can be used
+to approximate the medial axis and, as the discretized points tend to infinity the diagram will start to converge to
+the medial axis.
+
+![2-secting](images/voronoi-2-secting.png)
+![10-secting](images/voronoi-10-secting.png)
+
+It is quite evident in the above plots how the Voronoi diagram becomes much closer to the medial axis as we go from
+2-secting (bisecting) to 10-secting the edges of the shape.
+
+## Usage
+
+This library can be installed from NPM.
+
+```bash
+npm install max-inscribed-circle
+```
+
+To run it, simply import it and pass it a GeoJSON Polygon or MultiPolygon.
+
+```javascript
+import maxInscribedCircle from 'max-inscribed-circle';
+const polygon = {
+    "type": "Feature",
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [[
+            [0.0,0.0],
+            [1.0,0.0],
+            [1.0,3.0],
+            [2.0,3.0],
+            [2.0,0.0],
+            [3.0,0.0],
+            [3.0,4.0],
+            [0.0,4.0],
+            [0.0,0.0]
+        ]]
+    }
+};
+
+console.log(maxInscribedCircle(polygon));
+/*
+{
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [1.25,3.5]
+    },
+    "properties": {
+        "radius": 0.4994165362629234,
+        "units": "degrees"
+    }
+}
+*/
+```
+
+![Usage](images/usage.png)
+
+There are actually infinitely many maximum inscribed circles in this sample shape. This library returns the first one
+it finds. Additionally, due to floating point issues with JavaScript, the radius will not always be as accurate as we
+like. Mathematically we know the radius of the circle is 0.5 but we can see this is not what is returned by the library.
+
+### Coordinate Units
+
+It is important to note that due to the underlying `turf` dependencies, this library has been written to work primarily
+with `(lat,lon)` coordinates. If the polygon is in a known projection then it is recommended you transform it to
+`WGS84 (EPSG:4326)`. The `reproject` and `pro4` libraries on NPM are good for this.
+
+## Options
+
+There are three different options that can be specified when executing `maxInscribedCircle`. They are `numSegments`,
+`units`, and `decimalPlaces`. Each of these is documented in their own subsection below.
+
+### `numSegments` - default `2`
```

Comment:
Given how rough 2 is, and how nice 10 is, maybe the default should be 4 or 5? Just a thought, really, nothing to do with the user guide.

Reply:
I had wondered about that but didn't want to change it at this time. How does changing a default value fit into semantic versioning since it will change the default output with a new version?